src/services/identity/DeploymentClient.ts

```diff
@@ -88,6 +96,28 @@ class DeploymentClient {
       JEKYLL_ENV: branchName === "master" ? "production" : "staging",
     },
   })
+
+  generateUpdatePasswordInput = (
+    appId: string,
+    password: string
+  ): UpdateBranchCommandInput => {
+    if (password) {
+      return {
+        appId,
+        branchName: "staging",
```

Comment:
nit: prefer to have these shifted out cos they remain teh same

Reply:
Split into 2 methods following kishore's request: https://github.com/isomerpages/isomercms-backend/pull/806/commits/592ab87a469d24dd1564872dd7d79bfba4ee5c6b